nautobot/extras/tests/test_filters.py

```diff
@@ -252,25 +255,25 @@ def test_site(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
     def test_role(self):
-        device_roles = DeviceRole.objects.all()[:2]
+        device_roles = self.device_roles[:2]
         params = {"role_id": [device_roles[0].pk, device_roles[1].pk]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(device_roles))
```

Comment:
Probably not necessary to make this change but this should be refactored when ConfigContext gets a factory anyway.

Reply:
It was necessary because this one was vacillating before.